app/models/importers/content_items_by_organisation.rb

```diff
@@ -0,0 +1,20 @@
+module Importers
+  class ContentItemsByOrganisation
+    def run(slug)
+      organisation = Organisation.find_by(slug: slug)
```

Comment:
Previously, we used ```find_or_create_by``` for similar, if so then no need for this [comment](https://github.com/alphagov/content-performance-manager/pull/38/files#r94231710)